docs/source/alignment.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>Also hindcast experiments can be initialized in intervals, see the CMIP5 DCPP experiments. then we also cannot do all alignments. I would not force this alignment concept into the perfect model in the text here, because it just doesn't make sense in a perfect-model framework. I liked the statement you had in there earlier, something like: a perfect-model framework is perfectly aligned by design.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>